### DISCUSSION TITLE
Is there any seo Impact of dynamic ip in serverless frontend deployment ?

### DISCUSSION THREAD

**Discussion by rifat-h** [vercel/community#1969] [Help] [answered]
### Summary

Hello,
I am new to vercel and serverless technologies. As far as i know vercel will provide dynamic Ip on my domain.
Can anyone inform me more about how dynamic Ip will be assigned on the domain and its impact on seo.
Thanks.

**Comment by smaeda-ks** [comment, marked as answer]
I'm not an expert in that area either, but I believe resolved IPs don't matter for SEO these days anymore:
https://www.searchenginejournal.com/ranking-factors/is-ip-address-a-google-ranking-factor/

If you can share where exactly you're getting such concerns from (who/what links are suggesting that?), we're happy to take a look.

**Comment by calexjack234** [comment]
The use of a dynamic IP in serverless frontend deployment can potentially have an impact on SEO, but it is not necessarily a determining factor.

Search engines generally prioritize content relevance, quality, and user experience over technical infrastructure details such as IP addresses. However, a dynamic IP can lead to inconsistent accessibility and potentially affect website uptime, which can indirectly impact SEO by affecting user experience and engagement.

For instance, if a website like [Wolf Induction Cooktop Reviews](https://www.kitcheninduction.com/induction-wolf-cooktops/) has a dynamic IP, it may experience frequent downtime and connectivity issues, which can harm its SEO rankings. Search engines may perceive the website as unreliable, resulting in a lower ranking in search results.

Therefore, it is advisable to use a static IP address or a reliable cloud-based hosting service to ensure website availability and reliability. Additionally, optimizing website content, user experience, and engagement remains crucial for SEO success.

**Comment by betexkorin23** [comment]
The impact of a dynamic IP address on SEO for serverless frontend deployment depends on several factors, such as the frequency of IP address changes and the website's content and structure. In general, a dynamic IP address may affect SEO if search engines are unable to access the website due to IP address changes, resulting in indexing issues and decreased visibility. However, this can be mitigated by implementing best practices such as using a custom domain name, configuring DNS settings correctly, and monitoring IP address changes to ensure search engines can crawl the website.

For example, [Doubt Connect](https://www.doubtconnect.in/), a platform that connects people with experts or peers to get answers to their doubts or questions, could be impacted by IP address changes if search engines cannot access the website. However, by implementing SEO best practices, such as optimizing content and ensuring correct configuration of DNS settings and custom domain names, Doubt Connect can improve its visibility in search results and attract more users.
